### ISSUE TITLE
Write more tests

### ISSUE THREAD

**Issue by iriahi** [ahmia/search#24]
Tests need to be written for the django app and the crawler.
The coverage should be displayed on each project index.

**Comment by iriahi** [comment]
https://github.com/ahmia/search/issues/13 should be done first.

**Comment by iriahi** [comment]
I think we should postpone this as well, since we are going to refactor the django app, and change bots.

It would be good if travis.CI could test every django URLs (just like a user).
For bots, I read about [contracts](http://doc.scrapy.org/en/latest/topics/contracts.html).
